controllers/workspace/testdata/test-devworkspace-config.yaml

```diff
@@ -0,0 +1,29 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  labels:
+    controller.devfile.io/creator: ""
+spec:
+  started: true
+  routingClass: 'basic'
+  template:
+    attributes:
+      controller.devfile.io/devworkspace-config:
+        name: devworkspace-operator-config
```

Comment:
Are you specifically trying to make sure this works with an external DWOC instead of the global DWOC? If so, it might be worthwhile to use a different name for the DWOC, since this is the name of the global DWOC.

Reply:
I wanted to use the global DWOC, I removed this .yaml in my latest commit